app/components/preprint-form-section.js

```diff
@@ -59,26 +62,26 @@ export default CpPanelComponent.extend(Analytics, {
     // Called when panel is toggled
     handleToggle() {
         // Prevent closing all views
-        let isOpen = this.get('isOpen');
+        const isOpen = this.get('isOpen');
         if (!isOpen) {
             if (this.get('allowOpen')) {
                 // Crude mechanism to prevent opening a panel if conditions are not met
                 this.get('metrics')
                     .trackEvent({
                         category: 'div',
                         action: 'click',
-                        label: `${this.get('editMode') ? 'Edit' : 'Submit'} - Click to edit, ${this.name} section`
+                        label: `${this.get('editMode') ? 'Edit' : 'Submit'} - Click to edit, ${this.name} section`,
                     });
                 this._super(...arguments);
             } else {
-                this.sendAction('errorAction', this.get('denyOpenMessage'));
+                this.sendAction('errorAction', this.get('denyOpenMessage')); /* eslint-disable-line ember/closure-actions */
```

Comment:
@adlius the closure actions rule still applies here. The action should be called directly instead of bubbled: `this.get('errorAction')(this.get('denyOpenMessage');`

See [Ember Closure Actions in Depth](http://miguelcamba.com/blog/2016/01/24/ember-closure-actions-in-depth/) or [Ember Best Practices Stop Bubbling and use Closure Actions](https://dockyard.com/blog/2015/10/29/ember-best-practice-stop-bubbling-and-use-closure-actions)